lsp/io.ml

```diff
@@ -68,9 +88,31 @@ let logMessage fmt ~lvl ~message =
   let msg = Base.mk_notification ~method_ ~params in
   send_json fmt msg
 
-let logTrace fmt ~message ?verbose () =
+let logTrace fmt ~verbose ~message =
   let method_ = "$/logTrace" in
-  let verbose = Option.cata (fun v -> [ ("verbose", `String v) ]) [] verbose in
-  let params = [ ("message", `String message) ] @ verbose in
-  let msg = Base.mk_notification ~method_ ~params in
-  send_json fmt msg
+  if !trace_value = TraceValue.parse verbose then
```

Comment:
@artagnon I don't understand how this if is supposed to work, so `log_error` will produce no trace when `!trace_value = Verbose` ?